src/main/pages/che-7/overview/proc_finding-che-cluster-url-using-openshift-4-cli-tools.adoc

```diff
@@ -1,22 +1,33 @@
 [id="finding-{prod-id-short}-cluster-url-using-openshift-4-cli-tools_{context}"]
-= Finding {prod-short} cluster URL using OpenShift 4 CLI tools
+= Finding {prod-short} cluster URL using the OpenShift 4 CLI
 
-This section describes how to {prod-short} cluster URL using OpenShift 4 CLI tool.
+This section describes how to obtain the {prod-short} cluster URL using the OpenShift 4 CLI (command line interface). The URL can be retrieved from the OpenShift logs or from the `checluster` Customer Resource.
 
-[discrete]
-== Prerequisites
+.Prerequisites
+* {prod-short} is deployed in a OpenShift cluster. This can be validated by osbserving Openshift logs. See xref:viewing-the-state-of-the-{prod-id-short}-cluster-deployment-using-openshift-4-cli-tools_{context}[].
+* User is located in a {prod-short} installation namespace.
 
-* View the logs of the {prod-short} Operator. See xref:viewing-the-state-of-the-{prod-id-short}-cluster-deployment-using-openshift-4-cli-tools_{context}[].
+.Procedure
 
-[discrete]
-== Procedure
+. For retrieving a {prod-short} cluster URL from the `checluster` CR, run the following command:
```

Comment:
For retrieving -> to retrieve

I'd say "the following command" is redundant in all of the steps.
 "To do something, run:" or simply "To do something: " is sufficient IMO.